images/virtualization-artifact/pkg/controller/vi/vi_controller.go

```diff
@@ -81,10 +81,13 @@ func NewController(
 
 	reconciler := NewReconciler(
 		mgr.GetClient(),
+		dvcr.ImageMonitorSchedule,
+		log,
```

Comment:
It is not a good solution to pass log here. logger for handlers is constructed during `controller.New` call and handlers get logger from the context.

We need to create a more common pattern for the CronSource, but for now consider not to change `NewReconciler` and add CronSource after `SetupController`. See dvcr_maintanance_controller.go for inspiration.